apps/drec-api/src/utils/types.ts

```diff
@@ -0,0 +1,20 @@
+import { ICertificate as IOriginalCertificate } from '@energyweb/origin-247-certificate';
+
+export type EICCode = string;
+
+export enum CertificateSourceType {
+  Generator = 'generator',
+  Battery = 'battery',
+}
+
+/**
+ * Certificate source allows to describe how the certificate was issued.
+ * It can be issued by various entities like battery or generator itself
+ */
+export interface ICertificateMetadata {
+  type: CertificateSourceType;
+  deviceIds: number[];
+  id?: string;
```

Comment:
What's that?

Reply:
the id will be the groupId and the deviceIds - the list of ids for devices belonging to the group

Reply:
So why it's optional

Reply:
Well, there are devices which don`t belong to a group, so I create a group with id: 0 and add them there but maybe this will change so I thought making it optional would be better.